CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/CommentUpdaterCli.java

```diff
@@ -32,33 +33,35 @@
     String getCommentSchema() { return getThird(); }
   }
 
-  public static void main(String[] args) throws Exception {
+  void init(CommentSolrDocumentFields docFields, CommentUpdaterSql updaterSql) throws Exception {
```

Comment:
Looks like this is doing a lot more than initialization.  Rename to "performUpdate()" or similar.